docs/docs/keymaps/behaviors/mouse-emulation.md

```diff
@@ -69,3 +75,127 @@ This example will send press of the fourth mouse button when the binding is trig
 ```
 &mkp MB4
 ```
+
+### Input Processors
+
+If you want to apply any [input processors](../input-processors/index.md#input-processors-overview) to `&mkp` you can do so by referencing `&mkp_input_listener`, e.g.:
+
+```dts
+&mkp_input_listener {
+    input-processors = <&zip_temp_layer 2 2000>;
+}
+```
+
+## Mouse Move
+
+This behavior sends mouse X/Y movement events to the connected host.
+
+### Behavior Binding
+
+- Reference: `&mmv`
+- Parameter: A `uint32` with 16-bits each used for vertical and horizontal max velocity.
+
+The following predefined values can be passed for the parameter:
+
+| Define       | Action     |
+| :----------- | :--------- |
+| `MOVE_UP`    | Move up    |
+| `MOVE_DOWN`  | Move down  |
+| `MOVE_LEFT`  | Move left  |
+| `MOVE_RIGHT` | Move right |
+
+Additionally, if you want to pass a different max speed than the default for the `MOVE_*` defines, custom X and Y velocity values can be passed with `MOVE_X` and `MOVE_Y`, e.g. `MOVE_X(100)` or `MOVE_Y(-100)`.
+
+### Examples
+
+The following will send a down mouse movement event to the host when pressed/held:
+
+```
+&mmv MOVE_DOWN
+```
+
+The following will send a left mouse movement event to the host when pressed/held:
+
+```
+&mmv MOVE_LEFT
+```
+
+### Input Processors
+
+If you want to apply any [input processors](../input-processors/index.md#input-processors-overview) to `&mmv` you can do so by referencing `&mmv_input_listener`, e.g.:
+
+```dts
+&mmv_input_listener {
+    input-processors = <&zip_temp_layer 2 2000>;
+}
+```
+
+## Mouse Scroll
+
+This behavior sends vertical and horizontal scroll events to the connected host.
+
+### Behavior Binding
+
+- Reference: `&msc`
+- Parameter: A `uint32` with 16-bits each used for vertical and horizontal velocity.
+
+The following defines can be passed for the parameter:
+
+| Define       | Action       |
+| :----------- | :----------- |
+| `SCRL_UP`    | Scroll up    |
+| `SCRL_DOWN`  | Scroll down  |
+| `SCRL_LEFT`  | Scroll left  |
+| `SCRL_RIGHT` | Scroll right |
+
+Additionally, if you want to pass a different max speed than the default for the `SCRL_*` defines, custom X and Y velocity values can be passed with `MOVE_X` and `MOVE_Y`, e.g. `MOVE_X(5)` or `MOVE_Y(-5)`.
+
+### Examples
+
+The following will send a scroll down event to the host when pressed/held:
+
+```
+&msc SCRL_DOWN
+```
+
+The following will send a scroll left event to the host when pressed/held:
+
+```
+&msc SCRL_LEFT
+```
+
+:::note
+
+If you enabled [smooth scrolling](../../config/pointing.md#kconfig) then you will want to use the same `MOVE_UP`, `MOVE_DOWN`, etc values instead of the smaller `SCRL_*` parameters.
```

Comment:
```suggestion
If you enabled [smooth scrolling](../../config/pointing.md#kconfig) then you will want to use the same `MOVE_UP`, `MOVE_DOWN`, etc values instead of the smaller `SCRL_*` parameters for more sensible scroll speeds.
```